### PR TITLE
styling / layout changes for the resource picker

### LINKED ISSUE #620: style and design updates to the resource picker
see https://projects.invisionapp.com/share/NB11D8TFZX28#/screens/457070098_Resources_Widget_-_Inline_Form_Field

- [x] make modal bigger by default
- [x] remove 'filter' checkbox and always show filter textbox
- [x] add inline preview in a grid for images, videos
- [x] figure out some display for documents (documents should display as a list of items, somewhat like the site content on a site content page)
- [x] reorder tabs so that it is documents, videos, images

mockup screens:

Screen Shot 2021-09-29 at 11 57 11 AM
Screen Shot 2021-09-29 at 11 57 23 AM

Screen Shot 2021-09-29 at 11 57 32 AM

### PR DESCRIPTION
**Pull request by alicewriteswrongs**
#### Pre-Flight checklist

- [ ] Screenshots and design review for any changes that affect layout or styling
  - [ ] Desktop screenshots
  - [ ] Mobile width screenshots
- [ ] Testing
  - [ ] Code is tested
  - [ ] Changes have been manually tested

#### What are the relevant tickets?

closes #620

#### What's this PR do?

This makes some styling and layout tweaks to the resource picker. In particular, it

- makes the modal a little bigger
- removes the 'filter' checkbox in favor of always showing the filter input
- changes images and videos to a grid-based 'gallery' layout and polishes up the list of document titles as well
- reorders tabs

See the issue for details.

#### How should this be manually tested?

Try out the resource picker. It should work as you expect, and the styling should be a little nicer than before. You should see thumbnails for youtube videos, and you should also see a tiny version of any image you upload.

#### Screenshots (if appropriate)

Screen Shot 2021-10-04 at 3 11 00 PM
Screen Shot 2021-10-04 at 3 11 05 PM
Screen Shot 2021-10-04 at 3 13 39 PM
Screen Shot 2021-10-04 at 3 13 43 PM

### CODE REVIEW

File: static/scss/site-page.scss [hunk 1]
```diff
@@ -74,45 +74,3 @@ $menu-spacing: 20px;
...
-    .nav-item {
-      margin-left: 10px;
-      margin-right: 2px;
-      background-color: $dialog-gray;
-      border-top: 1px solid $dialog-gray;
-      border-left: 1px solid $dialog-gray;
-      border-right: 1px solid $dialog-gray;
-
-      &.active {
-        //border-bottom: white;
```
Comment (gumaerc) [inline static/scss/site-page.scss:113]: Should this be removed as well?

**Review by gumaerc — CHANGES_REQUESTED** [review]
This all looks good and functions well, I just noticed a few spots where I think maybe you meant to remove some styles?

File: static/scss/resource-picker-dialog.scss [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+    .nav-item {
+      margin-left: 10px;
+      margin-right: 2px;
+      background-color: $dialog-gray;
+      border-top: 1px solid $dialog-gray;
+      border-left: 1px solid $dialog-gray;
+      border-right: 1px solid $dialog-gray;
+
+      &.active {
+        //border-bottom: white;
```
[the remark this answers is not in the document]
Comment (alicewriteswrongs) [reply static/scss/resource-picker-dialog.scss:81]: I think that was commented out already, so I definitely will!

**Review by gumaerc — APPROVED** [review]
:+1: LGTM

**Pull request merged**